README_DEV.md

```diff
@@ -1,16 +1,55 @@
-## Development
-### Android Studio
+# Development
+## Android Studio
 This project was developed using Android Studio Electric Eel, but you can also use Android Studio Dolphin since it's currently using Android Gradle Plugin version 7.3.1.
 
-### How to run the unit tests
+## Get up and running locally
+1. You need to install [the Java Developer Kit (JDK)](https://www.oracle.com/java/technologies/downloads/). The JDK is what powers Android Studio ability to build and run Android apps. Technically, our CI/CD builds Forage Android SDK using [JDK 11](https://github.com/teamforage/forage-android-sdk/blob/f2ff61ed27847b28d631f975c9a79e52f9258802/.github/workflows/CI.yaml#L21). However, as of this writing, the latest stable version is [JDK 17](https://www.oracle.com/java/technologies/downloads/#java17).
+    > ⚠️ NOTE: Installing the Java Developer Kit (JDK) is **not** the same thing as “[installing Java on your machine](https://www.oracle.com/ca-en/java/technologies/downloads/)”. The latter refers to installing the Java Runtime Environment (JRE), which is what you would install to run a Java application. As we are developing Java (Android) applications, we need the JDK, which comes bundled with the JRE as well as other software facillities needed for Java development.
```

Comment:
Change facillities to facilities.

Change the sentence "As we are developing Java (Android)" to "As we are developing Kotlin (Android)".

Reply:
nice catch! Resolved in [`43eb5f8`](https://github.com/teamforage/forage-android-sdk/pull/44/commits/43eb5f84ce33cd137f8e2763aff0fe60edecf8b1)